app/models/drupal_user.rb

```diff
@@ -69,13 +72,20 @@ def unban
     self.status = 1
     increase_likes_unbanned
     save
+    update_user_status(1)
     self
   end
 
   def email
     mail
   end
 
+  def update_user_status(status)
+    u = user
```

Comment:
Could we make this `self.user` just to clarify a bit?

Reply:
Sure thing!!